src/main/java/uk/gov/hmcts/reform/professionalapi/controller/request/UserProfileCreationRequest.java

```diff
@@ -25,7 +25,8 @@
     private String email;
 
     @JsonIgnore
-    private static final String EMAILREGEX = "^[a-zA-Z0-9_!#$%&’*+/=?`{|}~^-]+(?:\\.[a-zA-Z0-9_!#$%&’*+/=?`{|}~^-]+)*@[a-zA-Z0-9-]+(?:\\.[a-zA-Z0-9-]+)*$";
+    private static final String EMAILREGEX = "^[a-zA-Z0-9_!#$%&’*+/=?`{|}~^-]+(?:\\.[a-zA-Z0-9_!#$%&’*+/=?`{|}~^-]+)"
+            + "*@[a-zA-Z0-9-]+(?:\\.[a-zA-Z0-9-]+)*$";
```

Comment:
Consider making this String into a Constant so it can be used in this class, `OrganisationCreationRequestValidator` and wherever else - also please use `.concat()` as you have done in `OrganisationCreationRequestValidator`